File: Pertemuan1.2-CPP/Buku.cpp
#include <iostream>
#include <string>

using namespace std;

class Buku
{
private:
    string judul;
    string pengarang;

public:
    Buku()
    {
    }

    Buku(string j, string p)
    {
        judul = j;
        pengarang = p;
    }

    void setJudul(string j)
    {
        judul = j;
    }

    void setPengarang(string p)
    {
        pengarang = p;
    }

    string getJudul()
    {
        return judul;
    }

    string getPengarang()
    {
        return pengarang;
    }

    ~Buku()
    {
    }
};

File: Pertemuan1.2-Java/Buku.java
class Buku {
    private String judul;
    private String pengarang;

    Buku() {
    }

    Buku(String j, String p) {
        this.judul = j;
        this.pengarang = p;
    }

    void setJudul(String j) {
        this.judul = j;
    }

    void setPengarang(String p) {
        this.pengarang = p;
    }

    String getJudul() {
        return this.judul;
    }

    String getPengarang() {
        return this.pengarang;
    }
}

class Main {
    public static void main(String[] args) {
        Buku b1;
        Buku b2;

        b1 = new Buku();
        b1.setJudul("J2ME");
        b1.setPengarang("Orang_1");
        System.out.println(b1.getJudul());
        System.out.println(b1.getPengarang());

        b2 = new Buku("J2EE", "Orang_2");
        System.out.println(b2.getJudul());
        System.out.println(b2.getPengarang());
    }
}